migrations/versions/d7c80ade926a_.py

```diff
@@ -1,7 +1,7 @@
 """empty message
 
 Revision ID: d7c80ade926a
-Revises: c2f28eca200f
```

Comment:
Heroku's migrations didn't seem very happy with this migration, and tracing it back, it seems it was dropping indices that didn't exist yet (they're created in this file).

I've tried re-running the migrations with this migration removed and things look okay – let me know if this is an issue?

Reply:
Let's try it and I'll deploy a fix if it breaks anything :)